### PR TITLE
Improved display of delete event functionality, refine workflow of deleting events

### PR DESCRIPTION
**Pull request by bp323**

### CODE REVIEW

**Comment by bp323** [comment]
Much improved workflow when deleting/updating/creating events, added some animations as well.

File: apps/timetable/admin/ui/css/admin.css [hunk 1]
```diff
@@ -411,6 +411,10 @@
     margin-top: -10px;
 }
 
+.gh-event-delete button {
+    width: 100%;
```
Comment (Coenego) [inline apps/timetable/admin/ui/css/admin.css]: The colour of the button might need to change to black to be in line with the admin UI design

**Comment by Coenego** [comment]
Left one small comment. Reassigning to @bp323 for follow-up

**Comment by bp323** [comment]
Followed up on review, back to @Coenego

**Pull request merged**